examples/contracts/aggregator-consumer/contracts/Price_Consumer.cairo

```diff
@@ -0,0 +1,84 @@
+%lang starknet
```

Comment:
why is this file named `Price_Consumer.cairo`?

Reply:
@qperrot

Reply:
I had to do the same implementation that you did here https://docs.chain.link/docs/l2-sequencer-flag/ so I keep the same names.

Reply:
@cds95

Reply:
ok then we should rename it to `PriceConsumerWithSequencerCheck` to match the example then.

Reply:
Done